### PR TITLE
Update bzlmod migration to mention enable_bzlmod

### LINKED ISSUE #19561: Bzlmod /external/migration page does not mention --enable_bzlmod
### Page link:

https://bazel.build/external/migration

### Problem description (include actual vs expected text, if applicable):

The need for the `--enable_bzlmod` flag is mentioned only once in the [bzlmod overview page](https://bazel.build/external/overview).  It's easy to miss this which means you spend a while figuring out why none of the Bzlmod sample content is working for you. 

At a minimum tutorials like the migration guide should mention it, but in general the flag and how to use it need to much more prominent in the documentation and general examples.  For example [this external guide](https://blog.aspect.dev/bzlmod#heading-kicking-the-tires-yourself) mentions setting the flag in your RC file as the first step before doing anything else.

### Where do you see this issue? (include link to specific section of the page, if applicable)

The https://bazel.build/external/migration is missing information on using the flag.

### Any other information you'd like to share?

Here is a proposed documentation patch. I can make this a PR if desired:
https://github.com/bazelbuild/bazel/compare/master...jlisee:bazel:jlisee/bzlmod-flag-migration-guide

### PR DESCRIPTION
**Pull request by jlisee**
It the migration guide make sure to explicitly tell the user to update their `.bazelrc` with the `--enable_bzlmod` so that Bzlmod will be active.  This resolves #19561

### CODE REVIEW

**Comment by keertk** [comment]
Hi @jlisee could you sign the CLA please?

File: site/en/external/migration.md [hunk 1]
```diff
@@ -32,6 +32,19 @@ your workspace root, maybe with comments like:
     # See MODULE.bazel for external dependencies setup.
     ```
 
+### Enable Blzmod in your bazelrc {:#enable-bzldmod}
+
+`.bazelrc` lets you set flags that apply every time your run Bazel.  To enable
+Bzlmod we use the `--enable_bzlmod` flag, and apply it to the `common` command so
```
Comment (Wyverald) [inline site/en/external/migration.md:38]: "To enable Bzlmod, use the" would be better. We tend to avoid using "we" (hah) in docs.

Comment (jlisee) [reply site/en/external/migration.md]: All fixed, thanks for the correction.

**Review by meteorcloudy — APPROVED** [review]
Thanks!

**Pull request closed without merging**